### PR TITLE
Update the `bal` Command in BBEs

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Update the `bal` command in BBEs.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_service.out [hunk 1]
```diff
@@ -8,4 +8,4 @@
 ballerina build service
```
Comment (ThisaruGuruge) [inline examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_service.out:8]:
```suggestion
bal build service
```

File: examples/threads-and-strands/threads_and_strands.description [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// A strand is a lightweight thread construct. There are two situations in which the Ballerina runtime creates a new strand:
+// A strand is a lightweight thread construct. There are two situations in which the bal runtime creates a new strand:
```
Comment (ThisaruGuruge) [inline examples/threads-and-strands/threads_and_strands.description:1]: I this change correct?

File: examples/threads-and-strands/threads_and_strands.bal [hunk 3]
```diff
@@ -52,7 +52,7 @@ public function case4() {
     io:println("After the wait action\n");
 }
 
-// Create two new strands. The Ballerina runtime assigns both strands to the same thread executing the current strand.
+// Create two new strands. The bal runtime assigns both strands to the same thread executing the current strand.
```
Comment (ThisaruGuruge) [inline examples/threads-and-strands/threads_and_strands.bal:55]: Invalid change?

**Comment by ThisaruGuruge** [comment]
Shall we go through the changes once more? Seems like just a `find and replace` command issued. There maybe places where the change is not valid, and also in some places the change is missing.

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_service.out (hunk 1 above)
Comment (praneesha) [reply examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_service.out]: Done.

File: examples/threads-and-strands/threads_and_strands.description (hunk 2 above)
Comment (praneesha) [reply examples/threads-and-strands/threads_and_strands.description]: Fixed in all occurrences.

File: examples/threads-and-strands/threads_and_strands.bal (hunk 3 above)
Comment (praneesha) [reply examples/threads-and-strands/threads_and_strands.bal]: Fixed in all occurrences.

**Comment by praneesha** [comment]
> Shall we go through the changes once more? Seems like just a `find and replace` command issued. There are places where the change is not valid, and also in some places the required changes are missing.

Done.

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**